### ISSUE TITLE
Make docs page more presentable

### ISSUE THREAD

**Issue by hari** [codse/animata#176]

**Comment by hari** [comment]
Some amazing docs sites:
1. https://docs.expo.dev
2. https://lens.xyz/docs
3. https://figure.ai/ - Love the header design on mobile

**Comment by SEEDART007** [comment]
can i work on this??

**Comment by SEEDART007** [comment]
@hari can i work on this??

**Comment by hari** [comment]
@SEEDART007 yes but this is out of the scope of hacktoberfest as it is a big one.

**Comment by SEEDART007** [comment]
> @SEEDART007 yes but this is out of the scope of hacktoberfest as it is a big one.

I just want to contribute I'm tired of these events..

**Comment by hari** [comment]
Assigned to you @SEEDART007 

Please share what changes you'd like to make before writing the code. We can discuss it first.

**Comment by SEEDART007** [comment]
@hari  besides building up the docs i want to say that i am using you website heavily on my recent projects......everything is pretty fine also the docs....but i find some difficulties , first one is that your website lacks input and textarea ui components as well as forms....can we together add those to your website?.....I am heavily dependent on your website for ui components but for these tiny little things i have to surf more ui website which is pretty bad.

**Comment by hari** [comment]
@SEEDART007 We can come up with plan and create a new `form` section for this.

**Comment by SEEDART007** [comment]
> @SEEDART007 We can devise a plan and create a new `form` section for this.

sure
